### PR TITLE
[stable8] Use APCu only if available in version 4.0.6 and higher

### PR DESCRIPTION
**Pull request by LukasReschke**
Backport of https://github.com/owncloud/core/pull/14273 as per https://github.com/owncloud/core/pull/14273#issuecomment-74576533

### CODE REVIEW

**Comment by karlitschek** [comment]
:+1:

**Comment by LukasReschke** [comment]
@owncloud-bot Retest this please

File: settings/templates/admin.php [hunk 1]
```diff
@@ -143,9 +143,24 @@
 	</p>
 
 </div>
+
 <?php
 }
 
+// Windows Warning
```
Comment (nickvergessen) [inline settings/templates/admin.php]: ??? copy paste fail?

**Comment by nickvergessen** [comment]
:+1: apart from the fail comment

**Comment by DeepDiver1975** [comment]
@owncloud-bot retest this please :gun:

**Comment by LukasReschke** [comment]
@owncloud-bot Retest this please :bomb:

**Pull request merged**